sound/soc/sof/ipc3-topology.c

```diff
@@ -1594,6 +1594,17 @@ static int sof_ipc3_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 	if (ret < 0)
```

Comment:
Can you explain in commit this is needed to support 3rd and 4th pins of SDW with IPC3 and Intel platforms. Older topology files have not used these pins, so this has not been an issue before (people will want to understand why this was not needed before and why this is needed now).

Reply:
Commit message updated.